packages/contracts-bedrock/src/L1/L2OutputOracle.sol

```diff
@@ -103,8 +105,10 @@ contract L2OutputOracle is Initializable, ISemver {
 
         startingTimestamp = _startingTimestamp;
         startingBlockNumber = _startingBlockNumber;
-        proposer = _proposer;
-        challenger = _challenger;
+        systemConfig = _systemConfig;
+
+        // Delete the proposer value which is no longer stored in this contract.
```

Comment:
Are there spacers that we haven't removed in the past?

I can remove this, but my inclination is that we should lean towards doing it when we have an opportunity.

Reply:
I am fine with leaving it

Reply:
Yes there are spacers in the xdm and bridge that we didn't remove before

Reply:
Right, IIRC not removing them was mostly a decision meant to limit scope creep. 
In this can we can easily fit it in with this PR. 

That said, there is also the option of removing the spacer entirely, and setting it to zero with `Storage.setUint(5,0)`. 
I prefer the current implementation with the explicit spacer as it prevents us from making a mistake prior to this getting out the door. 

Going to resolve this, feel free to reopen if you like.